cmd/minikube/cmd/config/enable.go

```diff
@@ -64,7 +64,7 @@ var addonsEnableCmd = &cobra.Command{
 `, out.V{"profileArg": tipProfileArg})
 
 		}
-		if err != nil {
+		if err == nil {
```

Comment:
are we sure this was wrong before ? this code been arround long time ...maybe it was a odd logic but correct ?

Reply:
yeah, this bug was introduced in https://github.com/kubernetes/minikube/pull/12231